keyboards/kmini/readme.md

```diff
@@ -0,0 +1,13 @@
+Kmini keyboard firmware
+======================
+
+The Revo Kmini is a 65% keyboard with an additional two columns on the left. It is programmed with the 'KMAC key map', which only works on Windows and according to a [user](https://www.keebtalk.com/t/programming-revo-kmini/2107/7) it cannot map media controls or change the capslock key.
```

Comment:
Feel free to use my username here. ;)

And to clarify, you can change the capslock.  I'm not sure how the reset works on it, though (if it's looking at the keycode, or if it's looking at the matrix location),  

Also, I didn't see the media keys, so I'm not sure about those

Reply:
I ported this for a friend of mine, he claimed that he wasn't able to change the capslock key, even though he set it to control. Anyways, the reset function is hardwired and should not depend on the keymap. I don't know if you ported the kmac, but if you did, thanks! You helped a lot with this port 👍

Reply:
Not a problem.  And no it wasn't me.

THough, looks like there is an issue with the board though, at least for the QMK configurator. 

And good to know, on the rest